torch/fft/__init__.py

```diff
@@ -93,6 +93,110 @@
     tensor([0.+0.j, 1.+0.j, 2.+0.j, 3.+0.j])
 """)
 
+fft2 = _add_docstr(_fft.fft_fft2, r"""
+fft2(input, s=None, dim=(-2, -1), norm=None) -> Tensor
+
+Computes the 2 dimensional discrete Fourier transform of :attr:`input`.
+
+Note:
+    :func:`~torch.fft.fft2` is a convenience alias for :func:`~torch.fft.fftn`
```

Comment:
Instead of making this a note, how about adding a second sentence to the preamble like: "Equivalent to ..." and linking fftn and showing how to map the arguments from this to fftn?

Reply:
@mruberry PTAL, I've changed it but am not entirely sure about "map[ping] the arguments from this to fftn". The arguments should be exactly the same, other than `dim` not accepting `None`.